server/model/properties.go

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"errors"
+	"fmt"
+)
+
+var ErrInvalidBoardBlock = errors.New("invalid board block")
+var ErrInvalidPropSchema = errors.New("invalid property schema")
+var ErrInvalidProperty = errors.New("invalid property")
+
+// Properties is a map of Prop's keyed by property id.
+type Properties map[string]Prop
```

Comment:
Probably we can find a less "generic" name for this, maybe `BlockProperties` and `BlockProp` later, or something like that. for me Properties can be any property of any object in the whole app.

Reply:
Fixed via https://github.com/wiggin77/focalboard/commit/4a4a1b5d6afa918a65c8a34d59ad93a7029a3300